### DISCUSSION TITLE
Preguntas en el live, como?

### DISCUSSION THREAD

**Discussion by sergiolopezloya** [LaunchX-InnovaccionVirtual/CursoIntroPython#67] [Q&A] [answered]
Hola, he tenido dudas desde que iniciamos, pero escribo el "preguntar" dentro del live o entro a menti.com pero no me salen mis preguntas. Como es que se interactua en el live?

**Comment by MSuarez0** [comment]
en el chat de Teams, solo que solo en tu chat saldrá tus dudas, pero ella vera el de todos, y cuando te resuelva algo, ella te contestara a ti como privado.

**Comment by doguedogue** [comment, marked as answer]
Es suerte, son muchas preguntas 😁

![image](https://user-images.githubusercontent.com/23409026/153365696-a5d3b707-305f-406a-80e5-33e94ae50eea.png)

**Reply by FernandaOchoa** [reply]
Tip: Mientras estoy explicando no veo el chat, pero al terminar los veo todos, si al terminar el live justo cuando estoy despidiendo la sesión me escriben, les contesto. Me quedo más o menos resolviendo dudas aún cuando terminó el live unos 10min.

**Reply by sergiolopezloya** [reply]
Ya vi, es eso, somos muchísimos, nos vemos mañana entonces ;)
